RNTester/rn-cli.config.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path')
+
+const config = {
```

Comment:
what is this?

Reply:
this file allows to override default configuration of the Metro packager

as you see, I defined two paths (roots), so we don't have to copy files (using `scripts/copy.js`) from the root directory each time we run the packager